EarthTables/model_defaults.hpp

```diff
@@ -0,0 +1,3 @@
+#include <string>
+const std::string MODEL_DIR = "/home/work/Desktop/AsymmetricEarth/EarthTables";
+const std::string PREM3D_DEFAULT = "/home/work/Desktop/AsymmetricEarth/EarthTables/prem_default.txt";
```

Comment:
You should not add paths like this, as it won't generalise to other users

Reply:
True... I didn't even think about that, since that's how it was in prem_defaults.hpp on my computer.  Is that a file that is configured when OscProb is installed?

Reply:
Yes, this file is produced by the Makefile. It shouldn't even exist in the repository. See here:

https://github.com/joaoabcoelho/OscProb/blob/423ef0fbf0a15951f833f1b5cc48a82f5ae61aab/Makefile#L70-L73

Reply:
OK.  I will remove it, then, modify EarthModelBinned.cxx to use prem_default.hpp, and add lines to the makefile to have it add variables for the EarthTables directory and the 3D version of prem_default.txt to prem_default.hpp.  Does that sound good to you?

Reply:
I finished making these changes...